release-notes.txt

```diff
@@ -0,0 +1,224 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-14
+
+! New in this release
+- OF-DPA: We now support creating and enabling KNET interfaces.
+- baseboxd:
+  - updated version to 1.11.1
+  - fixed several minor bugs related to bonds and terminaton MACs.  - some STP issues were fixed
```

Comment:
You probably missed a newline here. Might be worth spelling out what bugs/issues, or at least a bit less vague.

The STP issue was a major one, where initial STP state was sometimes missed, causing ports to forward/flood instead of block.

Reply:
fixed newline